wolfhsm/common.h

```diff
@@ -0,0 +1,226 @@
+
+/* common.h
```

Comment:
We already have `wh_common.h` I don't think we should add an additional common file

Reply:
fixed